README.md

```diff
@@ -252,6 +252,37 @@ When using _Retry_ the message can effectively cause starvation of other message
 until the message itself can be processed; on the other hand _Republish_ inserts the message to the original queue as a new message and it
 lets the consumer handle other messages (if they can be processed).
 
+#### Republishing
+Republishing is solved at application level with publishing a new message (with original content, params, etc.) to the original queue and
```

Comment:
```suggestion
Republishing is solved at application level with publishing a new message (with original content, headers, etc.) to the original queue and
```

Reply:
Not sure about this. "Params" is more general - headers, messageId, userId etc. :thinking:

Reply:
I have problem with the `params` word because it is not used in context of RabbitMQ clearly. Please try to replace with `headers, messageId, userId etc.` then.